src/pgmoneta_ext/utils.c

```diff
@@ -42,17 +45,19 @@
 #include <stdbool.h>
 
 int
```

Comment:
It would be better with an output parameter for the superuser role as a bool, and failure as the return value